### PR TITLE
[FR] Multi

### PR DESCRIPTION
**Pull request by Ganondorfosu**
French translation of osu! wiki - Multi page

### CODE REVIEW

File: wiki/Multi/fr.md [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+| Statut/Couleur | Description |
+| :-: | :-- |
+| **Normal/Bleue** | **Le joueur a *plus* de la moitié de sa vie**. La couleur du bleu deviendra plus claire si la  vie du joueur se rapproche du maximum et virera au rouge si la vie passe en dessous de la moitié de la barre de vie. |
+| **Danger/Violette-Rouge** | **Le joueur a *moins* de la moitié de sa vie**. Le rouge deviendra plus clair si la vie du joueur se rapproche de zéro et virera au bleu si la vie passe au dessus de la moitié de la barre de vie. |
+| **Échoué/Grise** | **Le joueur n'a plus de vie**. Le joueur peut continuer à jouer et si [Sudden Death (SD)](/wiki/Game_modifier/Sudden_Death)/[Perfect (PF)](/wiki/Game_modifier/Perfect) n'était pas utilisé, remplir la barre de vie entièrement ramènera le joueur à la vie et repassera au statut "Normal". Le score n'est pas ajouté au score de l'équipe tant que le joueur est dans ce statut. Le pseudo du joueur passera du blanc au rouge. |
+| **Tag/Verte** | *Tag Coop and Tag Team Vs seulement*. **C'est la couleur durant tout le match et ne changera pas selon la barre de vie**. Une flèche pointera le joueur actuellement en train de jouer avec une petite lueur blanche. |
```
Comment (Pachiru) [inline wiki/Multi/fr.md:311]:
```suggestion
| **Tag/Verte** | Disponible uniquement en *Tag Coop et Tag Team Vs*. **Cette couleur reste la même durant tout le match et ne change pas selon la barre de vie**. Une flèche pointe le joueur actuellement en train de jouer avec une petite lueur blanche. |
```

File: wiki/Multi/fr.md [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+| **Skipped/Blanche** | **Le joueur demande à passer l'introduction de la map si celle-ci en a une**. Les joueurs ayant appuyé sur le bouton `Skip` en bas à droite de l'écran auront une petite boîte jaune avec écrit "Skipped" en blanc en bas à droite de leur boîte. **Tous les joueurs doivent demander à passer l'introduction pour que celle-ci soit passée**. |
```
Comment (Pachiru) [inline wiki/Multi/fr.md:312]:
```suggestion
| **Skipped/Blanche** | **Le joueur demande à passer l'introduction de la map si celle-ci en a une**. Les joueurs ayant appuyé sur le bouton `Skip` en bas à droite de l'écran auront une petite rectangle jaune avec écrit "Skipped" en blanc en bas à droite de leur pseudo. **Tous les joueurs doivent demander à passer l'introduction pour que celle-ci soit passée**. |
```

File: wiki/Multi/fr.md [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+| **Quit/Dépend** | **Le joueur a quitté le match**. Deux possibilités d'avoir ce statut: (1) Appuyer sur `Echap` deux fois ou (2) se déconnecter de Bancho. La couleur du pseudo du joueur passera du blanc au rouge, avec un `[Quit]` écrit près le pseudo. La couleur de la boîte dépend de la vie qu'avait le joueur avant de quitter. |
```
Comment (Pachiru) [inline wiki/Multi/fr.md:313]:
```suggestion
| **Quit/Dépend** | **Le joueur a quitté le match**. Il y a deux possibilités d'obtenir ce statut: (1) Appuyer sur `Echap` deux fois ou (2) se déconnecter de Bancho. La couleur du pseudo du joueur passera du blanc au rouge, avec un `[Quit]` écrit près du pseudo. La couleur de la boîte dépend de la vie qu'avait le joueur avant de quitter la partie. |
```

File: wiki/Multi/fr.md [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+
+### Head to Head
+
+#### Interface
+
+![](img/Multi_HTH.jpg "Interface en Head to Head")
+
+Le Head to Head est un type de partie dans lequel les joueurs jouent chacun pour soi pour avoir le meilleur score de la salle.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:321]:
```suggestion
Le Head to Head est un type de partie dans lequel les joueurs jouent chacun pour soi pour obtenir le meilleur score de la salle.
```

File: wiki/Multi/fr.md [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+Il s'agit du type de partie par défaut et n'a pas de caractéristique spéciale, il s'agit simplement d'avoir le meilleur score.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:322]:
```suggestion
Il s'agit du type de partie par défaut et n'a pas de caractéristiques particuliers, il s'agit simplement d'avoir le meilleur score.
```

File: wiki/Multi/fr.md [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+#### Résultat
+
+![](img/Multi_grade.jpg "Ecran de résultat en Head to Head")
+
+A la fin du match, la photo de profil du joueur ayant atteint la première place sera affichée dans la partie "WINNER"
+
+Le rang obtenu est maintenant placé derrière le résultat.
+Le résultat des autres joueurs peut être observé en cliquant sur leurs boîtes respectives.
+
+Chaque joueur recevra le rang qu'il a obtenu au classement dans l'onglet privé `#userlog` de la console de chat.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:333]:
```suggestion
Chaque joueur reçoit dans l'onglet privé `#userlog` du chat le rang qu'il a obtenu au classement de chaque partie.
```

File: wiki/Multi/fr.md [hunk 7]
```diff
@@ -0,0 +1,429 @@
...
+
+### Team Vs
+
+#### Interface
+
+![](img/Multi_team_vs.jpg "Interface en Team Vs")
+
+Le mode Team Vs fait s'affronter deux équipes (Rouge et Bleue) selon la condition de victoire choisi par l'hôte.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:341]:
```suggestion
Le mode Team Vs fait s'affronter deux équipes (Rouge et Bleue) selon différentes conditions de victoire choisies par l'hôte.
```

File: wiki/Multi/fr.md [hunk 8]
```diff
@@ -0,0 +1,429 @@
...
+Remplissez la condition du mieux que vous le pouvez pour obtenir le rang le plus élevé dans votre équipe et battre le total de l'équipe adverse.
+
+Les membres de l'équipe bleue sont placés à gauche, les membres de l'équipe rouge sont placés à droite et une couronne au centre de l'écran montre quelle équipe domine.
+Elle bougera de gauche à droite selon l'équipe qui remplit le mieux la condition de victoire.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:345]:
```suggestion
Cette couronne bouge de gauche à droite selon l'équipe qui remplit le mieux la condition de victoire.
```

File: wiki/Multi/fr.md [hunk 9]
```diff
@@ -0,0 +1,429 @@
...
+
+Quand ce mode est sélectionné, tous les joueurs dans la salle recevront une couleur rouge ou bleue.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:347]:
```suggestion
Quand ce mode est sélectionné, tous les joueurs dans la salle reçoivent une couleur au hasard entre rouge ou bleue.
```

File: wiki/Multi/fr.md [hunk 10]
```diff
@@ -0,0 +1,429 @@
...
+Les joueurs peuvent cliquer dessus pour la changer et choisir entre rouge et bleu.
+
+Il est possible de commencer un match avec une seule équipe, et il n'y a pas de bonus pour l'équipe ayant moins de membres.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:350]:
```suggestion
Il est possible de commencer un match avec une seule équipe et il n'y a pas de bonus pour l'équipe ayant moins de membres.
```

File: wiki/Multi/fr.md [hunk 11]
```diff
@@ -0,0 +1,429 @@
...
+Le placement des joueurs avant le début du match ne change rien; tous les membres de l'équipe bleue seront regroupés dans l'équipe bleue et les membres de l'équipe rouge seront regroupés dans l'équipe rouge.
+Il n'y a pas de rôle (tel que "Leader" ou "Capitaine") dans les équipes; le but est simplement de s'affronter pour avoir le meilleur score de l'équipe et battre le score total de l'équipe adverse.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:352]:
```suggestion
Il n'y a pas de rôle (tel que "Leader" ou "Capitaine") dans les équipes; le but est simplement de s'affronter pour avoir le meilleur score de l'équipe et battre le score total adverse.
```

File: wiki/Multi/fr.md [hunk 12]
```diff
@@ -0,0 +1,429 @@
...
+
+Le membre survivant de chaque équipe ayant le meilleur score sera affiché comme `1er` dans son équipe.
+Les membres étant dans le statut Echoué seront placés en bas du classement de leur équipe et ne contribueront pas au total de leur équipe.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:355]:
```suggestion
Les membres étant dans le statut Échoué sont placés en bas du classement de leur équipe et ne contribuent pas au score total de leur équipe.
```

File: wiki/Multi/fr.md [hunk 13]
```diff
@@ -0,0 +1,429 @@
...
+Elle bougera de gauche à droite selon l'équipe qui remplit le mieux la condition de victoire.
+
+Quand ce mode est sélectionné, tous les joueurs dans la salle recevront une couleur rouge ou bleue.
+Les joueurs peuvent cliquer dessus pour la changer et choisir entre rouge et bleu.
+
+Il est possible de commencer un match avec une seule équipe, et il n'y a pas de bonus pour l'équipe ayant moins de membres.
+Le placement des joueurs avant le début du match ne change rien; tous les membres de l'équipe bleue seront regroupés dans l'équipe bleue et les membres de l'équipe rouge seront regroupés dans l'équipe rouge.
+Il n'y a pas de rôle (tel que "Leader" ou "Capitaine") dans les équipes; le but est simplement de s'affronter pour avoir le meilleur score de l'équipe et battre le score total de l'équipe adverse.
+
+Le membre survivant de chaque équipe ayant le meilleur score sera affiché comme `1er` dans son équipe.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:354]:
```suggestion
Le membre survivant de chaque équipe ayant le meilleur score est affiché comme `1er` dans son équipe.
```

File: wiki/Multi/fr.md [hunk 14]
```diff
@@ -0,0 +1,429 @@
...
+Les membres étant dans le statut Echoué seront placés en bas du classement de leur équipe et ne contribueront pas au total de leur équipe.
+
+#### Résultats
+
+![](img/Multi_team_grade.jpg "Ecran de résultats en Team Vs")
+
+Dans l'écran de résultat, le gagnant du match dépendra du total des équipes et est annoncé au milieu de l'écran avec la couleur de l'équipe gagnante. Le total de l'équipe bleue se trouve à gauche et le total de l'équipe rouge à droite.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:361]:
```suggestion
Dans l'écran de résultat, le gagnant du match dépend du score total des équipes et est annoncé au milieu de l'écran avec la couleur de l'équipe gagnante. Le total de l'équipe bleue se trouve à gauche et le total de l'équipe rouge à droite.
```

File: wiki/Multi/fr.md [hunk 15]
```diff
@@ -0,0 +1,429 @@
...
+
+**Seul le score des membres survivants sera compté**; il est possible que l'équipe rouge soit déclarée gagnante même si l'équipe bleue a plus de score parce que certains des joueurs de l'équipe bleue étaient dans le statut Echoué (ou tous les membres de l'équipe bleue ont échoué).
```
Comment (Pachiru) [inline wiki/Multi/fr.md:363]:
```suggestion
**Seul le score des membres survivants est compté**; il est possible que l'équipe rouge soit déclarée gagnante même si l'équipe bleue a plus de score parce que certains des joueurs de l'équipe bleue obtiennent le statut Échoué (ou si tous les membres de l'équipe bleue ont échoué).
```

File: wiki/Multi/fr.md [hunk 16]
```diff
@@ -0,0 +1,429 @@
...
+
+L'onglet `Total`  montrera le total des 300, 100, etc obtenus par toute l'équipe et la moyenne de précision de l'équipe.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:365]:
```suggestion
L'onglet `Total` montre le total des 300, 100, 50 et ratés obtenus par toute l'équipe ainsi que la moyenne de précision de l'équipe.
```

File: wiki/Multi/fr.md [hunk 17]
```diff
@@ -0,0 +1,429 @@
...
+Le combo de l'équipe ne sera pas compté dans l'onglet `Total` .
```
Comment (Pachiru) [inline wiki/Multi/fr.md:366]:
```suggestion
Le combo de l'équipe n'est pas compté dans l'onglet `Total` .
```

File: wiki/Multi/fr.md [hunk 18]
```diff
@@ -0,0 +1,429 @@
...
+Le rang obtenu ne sera pas affiché dans l'onglet  `Total` mais sera affiché affiché sur l'écran de résultat des joueurs (s'ils n'ont pas échoué).
```
Comment (Pachiru) [inline wiki/Multi/fr.md:367]:
```suggestion
Le rang obtenu n'est pas affiché dans l'onglet `Total` mais sera affiché sur l'écran de résultat des joueurs (s'ils n'ont pas échoué).
```

File: wiki/Multi/fr.md [hunk 19]
```diff
@@ -0,0 +1,429 @@
...
+
+*Seulement disponible dans le mode osu! standard. **NON-CLASSÉ***
+
+![](img/Multi_tag_co-op.jpg "Interface en Tag Coop")
+
+![](img/Multi_tag_team_vs.jpg "Interface en Tag Team Vs")
+
+Le Tag Coop fait travailler tous les joueurs en équipe pour réussir la beatmap, chaque joueur joue un combo chacun.
+
+Dans ce mode, le placement des joueur allant de haut en bas est important car il détermine l'ordre du tour des joueurs
```
Comment (Pachiru) [inline wiki/Multi/fr.md:381]:
```suggestion
Dans ce mode, le placement des joueurs allant de haut en bas est important car il détermine l'ordre du tour des joueurs
```

File: wiki/Multi/fr.md [hunk 20]
```diff
@@ -0,0 +1,429 @@
...
+
+Une flèche verte montrera quel joueur est actuellement en train de jouer le combo de la beatmap.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:383]:
```suggestion
Une flèche verte montre quel joueur est actuellement en train de jouer le combo de la beatmap.
```

File: wiki/Multi/fr.md [hunk 21]
```diff
@@ -0,0 +1,429 @@
...
+Le pseudo du joueur sera affiché en bas à droite de l'écran.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:384]:
```suggestion
Le pseudo du joueur est affiché en bas à droite de l'écran.
```

File: wiki/Multi/fr.md [hunk 22]
```diff
@@ -0,0 +1,429 @@
...
+Il y aura des flèches d'avertissements prévenant le joueur qu'il doit jouer et le combo sera en couleur jusqu'à ce que le combo se finisse
```
Comment (Pachiru) [inline wiki/Multi/fr.md:385]:
```suggestion
Il y aura des flèches d'avertissements prévenant le joueur qu'il doit jouer et le combo est en couleur jusqu'à ce que le combo se finisse
```

File: wiki/Multi/fr.md [hunk 23]
```diff
@@ -0,0 +1,429 @@
...
+Le joueur peut changer la couleur de son combo avant de commencer une map dans **Couleur**
+
+**Tous les joueurs doivent jouer les spinners**.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:388]:
```suggestion
**Tous les joueurs doivent jouer les spinners**. Ne pas jouer/réussir le spinner brisera le combo actuel.
```

File: wiki/Multi/fr.md [hunk 24]
```diff
@@ -0,0 +1,429 @@
...
+Ne pas jouer/réussir le spinner brisera le combo actuel.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:389]: Supprime cette ligne pour la remplacer au dessus.

File: wiki/Multi/fr.md [hunk 25]
```diff
@@ -0,0 +1,429 @@
...
+
+Si un joueur quitte durant le match, [Auto](/wiki/Game_modifier/Auto) jouera à sa place.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:391]:
```suggestion
Si un joueur quitte durant le match, le mode [Auto](/wiki/Game_modifier/Auto) prend le relais.
```

File: wiki/Multi/fr.md [hunk 26]
```diff
@@ -0,0 +1,429 @@
...
+
+En Tag Coop, tous les joueurs partageront la même barre de vie.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:393]:
```suggestion
En Tag Coop, tous les joueurs partagent la même barre de vie.
```

File: wiki/Multi/fr.md [hunk 27]
```diff
@@ -0,0 +1,429 @@
...
+Si la barre de vie est entièrement vidée (Echec), le match se termine et tous les joueurs sont renvoyés dans la salle immédiatement sans résultat
```
Comment (Pachiru) [inline wiki/Multi/fr.md:394]:
```suggestion
Si la barre de vie est entièrement vidée (Échec), le match se termine et tous les joueurs sont renvoyés dans la salle immédiatement sans résultat.
```

File: wiki/Multi/fr.md [hunk 28]
```diff
@@ -0,0 +1,429 @@
...
+
+En Tag Team Vs, chaque équipe a sa propre barre de vie qui n'est visible que par les mêmes membres d'une équipe.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:396]:
```suggestion
En Tag Team Vs, chaque équipe a sa propre barre de vie qui est visible uniquement par les membres de l'équipe.
```

File: wiki/Multi/fr.md [hunk 29]
```diff
@@ -0,0 +1,429 @@
...
+Si la barre de vie se vide entièrement, le match se termine et donne la victoire à l'équipe adverse.
+
+Si le mod [Easy (EZ)](/wiki/Game_modifier/Easy) est utilisé et que l'équipe échoue, les deux vies supplémentaires seront bien présentes mais les barres de vie seront immédiatement vidées afin d'échouer la beatmap.
+
+Les joueurs utilisant le mod [Sudden Death (SD)](/wiki/Game_modifier/Sudden_Death)/[Perfect (PF)](/wiki/Game_modifier/Perfect) auront une interface différente des autres joueurs, plus particulièrement quand il prend effet.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:401]:
```suggestion
Les joueurs utilisant le mod [Sudden Death (SD)](/wiki/Game_modifier/Sudden_Death)/[Perfect (PF)](/wiki/Game_modifier/Perfect) ont une interface différente des autres joueurs, plus particulièrement quand le mod prend effet.
```

File: wiki/Multi/fr.md [hunk 30]
```diff
@@ -0,0 +1,429 @@
...
+Si un coéquipier n'utilise **pas** ce mod et a fait une erreur suffisante pour causer l'effet du mod, la beatmap continuera normalement, et les joueurs utilisant le mod auront leur barre de vie réduite et bloquée à 0 sans causer l'effet du mod (seuls les effets visuels apparaîtront).
```
Comment (Pachiru) [inline wiki/Multi/fr.md:402]:
```suggestion
Si un coéquipier n'utilise **pas** ce mod et fait une erreur suffisante pour causer l'activation du mod, la beatmap continuera normalement et les joueurs utilisant le mod auront leur barre de vie réduite et bloquée à 0 sans causer l'effet du mod (seuls les effets visuels apparaîtront).
```

File: wiki/Multi/fr.md [hunk 31]
```diff
@@ -0,0 +1,429 @@
...
+Si le joueur **utilisant** le mod a fait une erreur suffisante pour causer l'effet du mod, celui-ci s'activera et l'échec sera immédiat.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:403]:
```suggestion
Si le joueur **utilisant** le mod fait une erreur suffisante pour causer l'activation du mod, celui-ci s'active et l'échec est immédiat.
```

File: wiki/Multi/fr.md [hunk 32]
```diff
@@ -0,0 +1,429 @@
...
+
+Si vous jouez en fenêtré, notez que le curseur ne sera **pas** confiné dans la fenêtre du jeu pendant les pauses ou pendant le tour des autres joueurs.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:405]:
```suggestion
Si vous jouez en fenêtré, notez que le curseur n'est **pas** confiné dans la fenêtre du jeu pendant les pauses ou pendant le tour des autres joueurs.
```

File: wiki/Multi/fr.md [hunk 33]
```diff
@@ -0,0 +1,429 @@
...
+
+#### Résultat
+
+![](img/Multi_co-op_grade.jpg "Ecran de résultat en Tag Coop")
+
+![](img/Multi_tag_team_grade.jpg "Ecran de résultat en Tag Team Vs")
+
+Comme précisé avant, le score ne sera pas compté pour le score classé et les points de performance car il s'agit d'une partie *non-classée*.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:413]:
```suggestion
Comme précisé plus haut dans l'article, le score n'est pas compté pour le score classé et les points de performance car il s'agit d'une partie *non-classée*.
```

File: wiki/Multi/fr.md [hunk 34]
```diff
@@ -0,0 +1,429 @@
...
+
+En Tag Coop, le résultat affiché est le score total, les 300, 100, etc et la précision moyenne de l'équipe
```
Comment (Pachiru) [inline wiki/Multi/fr.md:415]:
```suggestion
En Tag Coop, le résultat affiché comprend le score total, les 300, 100, 50 et ratés ainsi que la précision moyenne de l'équipe.
```

File: wiki/Multi/fr.md [hunk 35]
```diff
@@ -0,0 +1,429 @@
...
+Les résultats individuels ne seront pas montrés, et ne peuvent qu'être vus dans l'historique des matchs
```
Comment (Pachiru) [inline wiki/Multi/fr.md:416]:
```suggestion
Les résultats individuels ne sont pas montrés, et ne peuvent qu'être visionnés dans l'historique des matchs.
```

File: wiki/Multi/fr.md [hunk 36]
```diff
@@ -0,0 +1,429 @@
...
+L'écran de résultat n'apparaîtra que si la beatmap a été réussie.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:417]:
```suggestion
L'écran de résultat n'apparait que si la beatmap a été réussie.
```

File: wiki/Multi/fr.md [hunk 37]
```diff
@@ -0,0 +1,429 @@
...
+
+Pour l'écran de résultats en Tag Team Vs, voir l'explication de l'écran de résultats en Team Vs.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:419]:
```suggestion
*Pour l'écran de résultats en Tag Team Vs, voir l'explication de l'écran de résultats en Team Vs.*
```

File: wiki/Multi/fr.md [hunk 38]
```diff
@@ -0,0 +1,429 @@
...
+
+## Trivia
```
Comment (Pachiru) [inline wiki/Multi/fr.md:421]:
```suggestion
## Le Saviez-vous?
```

File: wiki/Multi/fr.md [hunk 39]
```diff
@@ -0,0 +1,429 @@
...
+
+### Historique
+
+![](img/Multi_Mania_unpatched.jpg "Capture d'écran d'une version non mise à jour d'osu! quand le mode de jeu osu!mania est sorti. \(8 Octobre 2012, 08/10/2012\)")
+
+- Les salles n'autorisaient qu'un maximum de 8 joueurs.
+- le mode de jeu osu!mania était noté `3` dans le lobby pour les joueurs qui n'avaient pas encore mis à jour leur jeu (qui active le mode osu!mania n'ayant pas encore de support de classement).
+- "No Video" était le seul mod autorisé pour les joueurs; cependant, ce mod a été supprimé en faveur de son placement dans les paramètres visuels.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:429]:
```suggestion
- Le mod "No Video" était le seul mod autorisé pour les joueurs; cependant, ce mod a été supprimé afin d'être directement intégré dans les paramètres visuels.
```

File: wiki/Multi/fr.md [hunk 40]
```diff
@@ -0,0 +1,429 @@
...
+Pour l'écran de résultats en Tag Team Vs, voir l'explication de l'écran de résultats en Team Vs.
+
+## Trivia
+
+### Historique
+
+![](img/Multi_Mania_unpatched.jpg "Capture d'écran d'une version non mise à jour d'osu! quand le mode de jeu osu!mania est sorti. \(8 Octobre 2012, 08/10/2012\)")
+
+- Les salles n'autorisaient qu'un maximum de 8 joueurs.
+- le mode de jeu osu!mania était noté `3` dans le lobby pour les joueurs qui n'avaient pas encore mis à jour leur jeu (qui active le mode osu!mania n'ayant pas encore de support de classement).
```
Comment (Pachiru) [inline wiki/Multi/fr.md:428]:
```suggestion
- Le mode de jeu osu!mania était noté `3` dans le lobby pour les joueurs qui n'avaient pas encore mis à jour leur jeu (qui active le mode osu!mania n'ayant pas encore de support de classement).
```

**Review by Pachiru — CHANGES_REQUESTED** [review]
Je pense qu'une fois que tout ça est appliqué, ce sera bon!

File: wiki/Multi/fr.md [hunk 41]
```diff
@@ -0,0 +1,428 @@
+---
+needs_cleanup: true
+---
+
+<!-- TODO: needs a thorough review for writing style and organisation, this is one of the more awkward articles on the site -->
+
+# Multi
+
+Le Multi (aussi connu en tant que *Multijoueur*) a été mis en œuvre le 3 Juin 2008 sous la version b335 d'osu!, et s'agit d'un mode de jeu dans le lequel jusqu'à 16 joueurs peuvent s'affronter sur une beatmap choisie par l'hôte.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:9]:
```suggestion
Le Multi (aussi connu en tant que *Multijoueur*) a été introduit le 3 Juin 2008 sous la version b335 d'osu!, et s'agit d'un mode de jeu dans lequel jusqu'à 16 joueurs peuvent s'affronter sur une beatmap choisie par l'hôte.
```

File: wiki/Multi/fr.md [hunk 42]
```diff
@@ -0,0 +1,428 @@
...
+
+## Lobby
+
+![](img/Multi_lobby.jpg "Exemple d'un jour typique dans le lobby")
+
+Le lobby affiche toutes les salles disponibles sur Bancho.
+
+En haut de l'écran se trouvent les filtres à gauche et le jukebox à droite.
+
+Les filtres sont expliqués ci-dessous:-
```
Comment (Pachiru) [inline wiki/Multi/fr.md:50]:
```suggestion
Les filtres sont expliqués ci-dessous:
```

File: wiki/Multi/fr.md [hunk 43]
```diff
@@ -0,0 +1,428 @@
...
+
+Au milieu sont affichées les salles disponibles après avoir été filtrées.
+
+Chaque salle contient ses informations séparées de gauche à droite pour faciliter la lecture:-
+
+- Du côté gauche, il y a le symbole du mode de jeu, le type de partie entre parenthèses, le nombre de joueurs présents dans la salle sur le nombre maximum de joueurs autorisé, et la différence de classement des joueurs allant du plus haut au plus bas
+- Du côté droite, il y a la photo de profil de l'hôte en grand et celles des autres joueurs en plus petit ainsi que le nom de la salle et la beatmap actuellement sélectionnée par l'hôte avec la difficulté notée entre crochets.
+  - La couleur rouge signifie que la place est prise (par un joueur).
+  - La couleur verte signifie que la place est disponible.
+  - Pas de couleur signifie que la place est verrouillée.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:72]:
```suggestion
  - Aucune couleur signifie que la place est verrouillée.
```

File: wiki/Multi/fr.md [hunk 44]
```diff
@@ -0,0 +1,428 @@
...
+
+## Nouvelle salle
+
+![](img/Multi_new_game.jpg "Configuration d'une nouvelle salle")
+
+| Titre | Description |
+| :-: | :-- |
+| **Game Name** | **Nom de la salle.** Par défaut appelée "`{Nom du compte}`'s game". Pour rendre votre salle privée, cliquez sur le bouton  `Require Password to join`. |
+| **Password** | **Mot de passe à entrer pour accéder à la salle** N'apparaît que si le bouton `Require Password to join` est activé. Le symbole du mode de jeu apparaîtra alors avec un cadenas. |
+| **Max players** | **Nombre de joueurs max. autorisé.** Par défaut à 8 joueurs, peut être de 2 à 16 joueurs, l'hôte de la partie étant inclus. Cela peut être changé après la création de la salle en déverrouillant/verrouillant les places dans la salle. |
```
Comment (Pachiru) [inline wiki/Multi/fr.md:102]:
```suggestion
| **Max players** | **Nombre de joueurs maximum autorisé.** Par défaut, le nombre autorise jusqu'a à 8 joueurs maximum, mais peut en réalité être modifié pour autoriser 2 à 16 joueurs, l'hôte de la partie étant inclus. Cela peut être changé après la création de la salle en déverrouillant/verrouillant les places dans la salle. |
```

File: wiki/Multi/fr.md [hunk 45]
```diff
@@ -0,0 +1,428 @@
...
+#### Beatmap
+
+![](img/Multi_host_song_change.jpg "Lorsque l'hôte est en train de changer de map, ce texte sera temporairement affiché.")
+
+![](img/Multi_SS.jpg "Exemple de ce que l'hôte voit en sélectionnant une map.")
+
+La partie du milieu est la partie Beatmap.
+
+Cette partie montre la beatmap sélectionnée par l'hôte.
+Celle-ci montrera l'image, le symbole du mode de jeu, le titre, l'artiste de la musique, le créateur de la beatmap, la difficulté sélectionnée ainsi que la difficulté en étoiles de la beatmap.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:167]:
```suggestion
Celle-ci montre l'image, le symbole du mode de jeu, le titre, l'artiste de la musique, le créateur de la beatmap, la difficulté sélectionnée ainsi que la difficulté en étoiles de la beatmap.
```

File: wiki/Multi/fr.md [hunk 46]
```diff
@@ -0,0 +1,428 @@
...
+
+![](img/Multi_mods_player.jpg "Options de mods \(osu! standard\) d'un joueur \(avec l'option "Mods libres" activée\)")
+
+La partie en bas à gauche est dédiée aux mods.
+Elle montre les [mods](/wiki/Game_modifier) utilisés pour ce match.
+
+L'hôte peut choisir d'activer les `Mods libres` afin de laisser les joueurs choisir les mods avec lesquels ils souhaitent jouer à l'exception des mods modifiant la vitesse ([Double Time (DT)](/wiki/Game_modifier/Double_Time)/[Nightcore (NC)](/wiki/Game_modifier/Nightcore) ou [Half Time (HT)](/wiki/Game_modifier/Half_Time)) de la beatmap.
+Les mods choisis par l'hôte n'affecteront pas les autres joueurs à l'exception des mods modifiant la vitesse.
+
+#### Type de partie, Condition de victoire, Couleur
```
Comment (Pachiru) [inline wiki/Multi/fr.md:191]:
```suggestion
#### Type de partie, Condition de victoire et Couleur
```

File: wiki/Multi/fr.md [hunk 47]
```diff
@@ -0,0 +1,428 @@
...
+
+Si un joueur quitte durant le match, le mode [Auto](/wiki/Game_modifier/Auto) prend le relais.
+
+En Tag Coop, tous les joueurs partagent la même barre de vie.
+Si la barre de vie est entièrement vidée (Échec), le match se termine et tous les joueurs sont renvoyés dans la salle immédiatement sans résultat.
+
+En Tag Team Vs, chaque équipe a sa propre barre de vie qui est visible uniquement par les membres de l'équipe.
+Si la barre de vie se vide entièrement, le match se termine et donne la victoire à l'équipe adverse.
+
+Si le mod [Easy (EZ)](/wiki/Game_modifier/Easy) est utilisé et que l'équipe échoue, les deux vies supplémentaires seront bien présentes mais les barres de vie seront immédiatement vidées afin d'échouer la beatmap.
```
Comment (Pachiru) [inline wiki/Multi/fr.md:398]:
```suggestion
Si le mod [Easy (EZ)](/wiki/Game_modifier/Easy) est utilisé et que l'équipe échoue, les deux vies supplémentaires seront bien présentes mais les barres de vie seront immédiatement vidées afin de terminer immédiatement la beatmap.
```

File: wiki/Multi/fr.md [hunk 48]
```diff
@@ -0,0 +1,428 @@
...
+*Pour l'écran de résultats en Tag Team Vs, voir l'explication de l'écran de résultats en Team Vs.*
+
+## Le Saviez-vous?
+
+### Historique
+
+![](img/Multi_Mania_unpatched.jpg "Capture d'écran d'une version non mise à jour d'osu! quand le mode de jeu osu!mania est sorti. \(8 Octobre 2012, 08/10/2012\)")
+
+- Les salles n'autorisaient qu'un maximum de 8 joueurs.
+- Le mode de jeu osu!mania était noté `3` dans le lobby pour les joueurs qui n'avaient pas encore mis à jour leur jeu (qui active le mode osu!mania n'ayant pas encore de support de classement).
```
Comment (Pachiru) [inline wiki/Multi/fr.md:427]:
```suggestion
- Le mode de jeu osu!mania était noté `3` dans le lobby pour les joueurs qui n'avaient pas encore mis à jour leur jeu (qui active le mode osu!mania qui n'avait pas encore de support de classement).
```

**Review by Pachiru — CHANGES_REQUESTED** [review]
Apply ça et normalement c'est bon

**Review by Pachiru — APPROVED** [review]
LGTM

**Review by TicClick — APPROVED** [review]
while I assume the translation is ok, I highly advise against translating the articles marked as outdated or in need of cleanup for obvious reasons (you'll have to update it once again when the original article is reworked)

**Pull request merged**